frontend/integration-tests/protractor.conf.ts

```diff
@@ -91,11 +91,14 @@ export const config: Config = {
     newApp: ['tests/base.scenario.ts', 'tests/overview/overview.scenario.ts', 'tests/source-to-image.scenario.ts', 'tests/deploy-image.scenario.ts'],
     olm: ['tests/base.scenario.ts', 'tests/olm/descriptors.scenario.ts', 'tests/olm/catalog.scenario.ts', 'tests/olm/prometheus.scenario.ts', 'tests/olm/etcd.scenario.ts'],
     olmUpgrade: ['tests/base.scenario.ts', 'tests/olm/update-channel-approval.scenario.ts'],
+    operatorHub: ['tests/base.scenario.ts', 'tests/operator-hub/operator-hub.scenario.ts'],
+    // OLM and OperatorHub
+    olmFull: ['tests/base.scenario.ts', 'tests/operator-hub/operator-hub.scenario.ts', 'tests/olm/descriptors.scenario.ts', 'tests/olm/catalog.scenario.ts', 'tests/olm/prometheus.scenario.ts', 'tests/olm/etcd.scenario.ts'],
```

Comment:
We don't need this suite; the OLM and Operator Hub suites should always be ran separately.

Reply:
The reason to group them would be to avoid a separate install for each suite when run as part of CI